### DISCUSSION TITLE
Onde eu vejo o dinheiro que entrou? DG Parnaíba

### DISCUSSION THREAD

**Discussion by emillyhorta** [apyb/comunidade#153] [Pergunta]
Tentei ver o dinheiro q entrou da RGM pro django girls Parnaíba mas n tem lá no balanço, só tem o de 2023. @cecivieira

https://financeiro.python.org.br/apyb/balance_sheet/?interval=week

O comprovante de pagamento 
[Pagto Assoc Python Brasil - 2024-08-14.pdf](https://github.com/user-attachments/files/17213529/Pagto.Assoc.Python.Brasil.-.2024-08-14.pdf)

**Comment by cecivieira** [comment]
Oi @Emilly-horta esse evento desejado só teve movimentação em agosto e o nosso site está atualizado até julho.
Acredito que ainda hoje as informações de agosto estarão no ar e você conseguirá acessá-la.

Acessa esse site amanhã ou a notie que você terá a informação.
